packages/cbioportal-frontend-commons/package.json

```diff
@@ -27,7 +27,7 @@
     "tcm": "tcm -p src/**/*.module.scss",
     "tcm:watch": "yarn run tcm --watch",
     "prepare": "yarn run build",
-    "test": "cross-env jest --env=jsdom --runInBand --ci --reporters=default --reporters=jest-junit",
+    "test": "cross-env jest $GREP --env=jsdom --runInBand --ci --reporters=default --reporters=jest-junit --passWithNoTests",
```

Comment:
these pass with no tests needed to be added in order to support GREP because there will be no matches there.